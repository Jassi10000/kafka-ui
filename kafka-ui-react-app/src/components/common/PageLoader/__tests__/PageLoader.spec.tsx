import { mount } from 'enzyme';
import React from 'react';
import PageLoader from 'components/common/PageLoader/PageLoader';

describe('PageLoader', () => {
  it('matches the snapshot', () => {
    expect(mount(<PageLoader />)).toMatchSnapshot();
  });
});
