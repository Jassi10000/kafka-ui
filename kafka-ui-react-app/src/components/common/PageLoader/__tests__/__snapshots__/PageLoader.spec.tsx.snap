// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageLoader matches the snapshot 1`] = `
<PageLoader>
  <styled.div>
    <div
      className="sc-gsTEea bVlHEY"
    >
      <styled.div>
        <div
          className="sc-bdfBQB iJLklD"
        />
      </styled.div>
    </div>
  </styled.div>
</PageLoader>
`;
